### PR TITLE
Fixed missing quotation marks when registering for push with mixpanel.

### PR DESCRIPTION
**Pull request by scanniza**
Fixed a bug when registering for Android push notifications. Mixpanel wants the pushId wrapped with quotes. This might be an error in the Mixpanel API but this is a fix anyways.

### CODE REVIEW

File: src/android/MixpanelPlugin.java [hunk 1]
```diff
@@ -229,7 +229,7 @@ private boolean handlePeopleSetOnce(JSONArray args, final CallbackContext cbCtx)
 
     private boolean handlePeopleSetPushId(JSONArray args, final CallbackContext cbCtx) {
         String pushId = args.optString(0);
-        mixpanel.getPeople().setPushRegistrationId(pushId);
+        mixpanel.getPeople().setPushRegistrationId('"'+pushId+'"');
```
Comment (samzilverberg) [inline src/android/MixpanelPlugin.java]: whaaaat why? :)
can't see any other examples floating around with that kind of fix.

Maybe its related to this issue on mixpanel-android github:
https://github.com/mixpanel/mixpanel-android/issues/227
this was fixed in 4.7.0

I'd rather update to 4.7.0 and try it again, if you got time soon you can try replacing the lib in the plugin dir and perhaps reference in plugin.xml.  
i'll try to check into it on the weekend if you dont get to it.

Comment (scanniza) [inline src/android/MixpanelPlugin.java]: You're absolutely right. I'll Do that tomorrow and send a new pull-request.
/Simon

Skickat från min iPhone

Comment (scanniza) [inline src/android/MixpanelPlugin.java]: Hey man, turns out Mixpanel stopped supplying .jars, but I suggest I will build my own?

Comment (samzilverberg) [inline src/android/MixpanelPlugin.java]: yup it needs to be built:
https://github.com/mixpanel/mixpanel-android/blob/v4.7.0/BUILDING.txt

**Pull request closed without merging**